samples/src/Microsoft.Azure.IIoT.App/src/Validation/PublisherInfoValidator.cs

```diff
@@ -14,15 +14,15 @@ public class PublisherInfoValidator : AbstractValidator<PublisherInfoRequested>
         public PublisherInfoValidator() {
             RuleFor(p => p.RequestedMaxWorkers)
                 .Must(BePositiveInteger)
-                .WithMessage("Max workers value cannot be less than 1.");
+                .WithMessage("Max workers must be an integer and cannot be less than 1.");
```

Comment:
Could we change "... cannot be less" to a positive sentence such us "... greater than", also to be consistent with the rest of the validation messages?